releasenotes/notes/fix-calling-OR-gate-as-AND-f1b364d04734098d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed the name of the `OrGate`, that was set to `"and"` instead of `"or"`, and could have possibly led to several problems.
```

Comment:
```suggestion
    Fixed the name of the :class:`.OrGate`, that was set to `"and"` instead of 
    `"or"`, and could have possibly led to several problems.
```